doc/getting_involved_in_mir.md

```diff
@@ -23,6 +23,8 @@ With these installed you can checkout Mir and get the remaining dependencies:
 
     $ git clone https://github.com/MirServer/mir.git
     $ cd mir
+    $ git submodule init
+    $ git submodule update
```

Comment:
Thanks, it'd be better to use:
`git submodule update --init --recursive`, in case our submodules get submodules.